diff --git a/node_modules/react-native-actions-sheet/index.d.ts b/node_modules/react-native-actions-sheet/index.d.ts
index c87a3c8..c6a8e1f 100644
--- a/node_modules/react-native-actions-sheet/index.d.ts
+++ b/node_modules/react-native-actions-sheet/index.d.ts
@@ -22,6 +22,9 @@ declare module "react-native-actions-sheet" {
      export const removeHasReachedTopListener: (callback: () => void) => void
 
      export type ActionSheetProps = {
+
+          children?: React.ReactNode;
+
           /**
            * Animate the opening and closing of ActionSheet.
       
